assets/presto_parquet/Dockerfile

```
@@ -0,0 +1,56 @@
+FROM presto/prestissimo-dependency:ubuntu-22.04
+
+# Install necessary packages
+RUN apt-get update;
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)